public/drawImageWorker.js

```diff
@@ -0,0 +1,46 @@
+
```

Comment:
Web workers need to be somewhere accessible for the frontend main thread to access them during exection, so this public dir works.